requirements/doc.txt

```diff
@@ -146,7 +146,7 @@ six==1.16.0
     #   sphinxcontrib-napoleon
 snowballstemmer==2.2.0
     # via sphinx
-sphinx==5.2.2
+sphinx==5.2.3
```

Comment:
No breaking change in the updated version https://www.sphinx-doc.org/en/master/changes.html#release-5-2-3-released-sep-30-2022